### ISSUE TITLE
train params

### ISSUE THREAD

**Issue by BigPuns** [Guillem96/spynet-pytorch#3]
hi, bro, I train spynet use your code by flyingchair dataset, set batchsize is 128, the loss in 24*32 is 4000+, in 384*512 is 5w+, is that correct？cannot get the ori optical flow result.
